ansible/configs/controller-migration-casc/sample_vars.yml

```diff
@@ -0,0 +1,190 @@
+#    Note down the first /hostedzone/XYZ123... ID (the one named like the sandbox domain without GUID).
+# 3. Call `aws ec2 import-key-pair --key-name id_rsa_xxx --region=us-west-1 --output=text \
+#              --public-key-material "$(cat ~/.ssh/id_rsa_xxx.pub)"`.
+
+cloud_provider: ec2                     # Which AgnosticD Cloud Provider to use
+aws_region: us-west-1                   # CHANGEME-ec2
+HostedZoneId: XYZ123ABX456R6            # CHANGEME-ec2
+ssh_provision_key_name: id_rsa_xxx      # CHANGEME-ec2
+
+aws_access_key_id: somethingcomplex               # CHANGEME-ec2
+aws_secret_access_key: somethingevenmore complex  # CHANGEME-ec2
+
+key_local_path:
+- "~/.ssh/{{ssh_provision_key_name}}.pem"
+
+#Ansible Tower related vars
+
+software_to_deploy: tower              # Define tower to install tower or none to have only infra ready.
+
+#For code-server installation on bastion
+vscode_server_hostname: "bastion.{{ guid }}{{ subdomain_base_suffix }}"
+vscode_user_password: changeme
+vscode_user_name: ec2-user
+
+# Dumps all vars to local dir /tmp/
+dump_vars: false
+
+#### Worker Node for Isolated Node group
+#worker_instance_type: "t2.medium"
+#worker_instance_count: 1              # Set 0 to not to provision worker(isolated) nodes.
+
+## target_region for worker regions
+target_regions:
+- name: emea
+#  - name: apac
+
+# CHANGEME (secrets)
+# e.g. http://example.com/ansible-automation-platform-setup-bundle-latest.tar.gz"
+deploy_automationcontroller_installer_url: ""
+
+# enable the development tools repo(s)? false by default because slow
+aap_devel_tools_repos_enable: false
+
+# install the development tools packages? true by default
+aap_devel_tools_install: true
+
+# not used with license injector role
+#deploy_automationcontroller_manifest_path: "~/secrets/automationcontroller_manifest.zip"
+
+#infra_workloads:
+#  - tower-copy-ssh                  ;;; specific to dark-tower
+#  - tower-settings-update
+#  - tower-pip-packages
+#  - tower-user-create
+#  - tower-custom-venv
+#  - tower-org-create
+#  - tower-credential-create
+#  - tower-project-create
+#  - tower-inventory-create
+#  - tower-job-template-create
+#  - tower-babylon-job-runner
+
+
+### tower project roles
+tower_projects:
+- name: darktower-dev
+  description: "babylon dev project"
+  organization: "gpte"
+  scm_url: "https://github.com/redhat-gpte-devopsautomation/dark-tower.git"
+  #scm_type:
+  #scm_credential:
+  scm_branch: dev
+  scm_update_on_launch: true
+
+tower_inventories:
+- name: empty-inventory-default
+  description: "Empty inventory default"
+  organization: gpte
+  # instance_group: ""
+
+#Define users you want to create. Set superuser: yes to make user system wide System Administrator
+tower_user_accounts:
+- user: babylon
+  password: changeme
+  email: babylon@example.com
+  firstname: Baby
+  lastname: Lon
+  superuser: true
+- user: babylon-viewer
+  password: changeme
```

Comment:
can this value be left empty in samples and examples and then the roles automatically generate a random password if that's the case?
We've seen too many hardcoded passwords and that becomes a problem